comid-triples.cddl

```diff
@@ -0,0 +1,142 @@
+concise-mid-tag = {
+  ; TODO(tho) language
+  ; TODO(tho) tag-id
+  ; TODO(tho) module-name
+  ; TODO(tho) entity
+  ; TODO(tho) linked-tags
+  comid.claims => claims-map
+  ; TODO(tho) extension
+}
+
+claims-map = non-empty<{
+  comid.reference-claims => one-or-more<reference-claim-record>
+  comid.endorsed-claims => one-or-more<endorsed-claim-record>
+  comid.identity-claims => one-or-more<identity-claim-record>
+  * $$claims-map-extension
+}>
+
+; REFERENCE-VALUE triple
+; "target environment $TE" "has reference measurements" "$RV"
+reference-claim-record = {
+  target-environment: environment-map 
+  reference-values: values-map
+}
+
+; ENDORSED-VALUE triple
+; "environment $E" "has endorsed measurements" "$EV"
+endorsed-claim-record = {
+  environment: environment-map 
+  endorsed-values: values-map
+}
+
+; DEVICE-IDENTITY triple
+; (Should it be AE-IDENTITY instead?)
+; "attesting environment $AE" "signs Evidence with key" "$K"
+identity-claim-record = {
+  attesting-environment: environment-map 
+  identity: COSE_Key
+}
+
+environment-map = non-empty<{
+  ? comid.class => class-map
+  ? comid.instance => instance-map
```

Comment:
Environment map should have extensibility in case we want to introduce group identifier or alias id.

Reply:
Again, this is 5b, where there is no multiplicity.  I also note that a COSE_Key is not limited to a single key, it could hold a complete cert chain.